Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM rust:1.53.0 as builder
+WORKDIR /usr/src
+RUN rustup target add x86_64-unknown-linux-musl
```

Comment:
why musl? If we end up running it within docker musl is not really useful here is it?

Reply:
```suggestion
```